state_daemon/src/main.rs

```diff
@@ -0,0 +1,51 @@
+/// The main entry point for the people_present daemon
+
+extern crate hyper;
+extern crate rustc_serialize;
+
+use rustc_serialize::json::{self, DecoderError};
+use hyper::Client;
+
+#[derive(RustcDecodable)]
+struct SwitchStatus {
+    open: bool,
+    people: u8,
+}
+
+fn update_people_temp_raspi(host: &str, people: u8, temp_raspi: f64) {
+    let mut client = Client::new();
+
+    let values =format!("people={}&temp_raspi={}", people, temp_raspi);
```

Comment:
I think a function to update values on the server could be implemented in the spaceapi_server crate. It would just take the hostname and a key value pair.
That way we can implement things like authentication in a single place.
Also I think it should be possible to update a single value at a time. Currently the python implementation requires one to send both.